### DISCUSSION TITLE
#25. VATZ biweekly meeting at 2023-01-04 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#346] [General]
### 1. Overall
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/6308023/209293562-1a5c0bca-9faa-4068-a1a8-48a75247bd2b.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
23 | 1/15 | 6
24 | 5/21 | 23

---
1. Stop command on CLI
   - Temp(CLIs - Stop)
   - or will be TBD - CLIs documents
    
2. Monitoring in seperate ...
   - Later End of Jan 2023

3. CLI documents
- Add Arguments in documents.

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/330
- https://github.com/dsrvlabs/vatz/issues/349 
- https://github.com/dsrvlabs/vatz/issues/351
- https://github.com/dsrvlabs/vatz/issues/352

**Comment by heejin-github** [comment]
## 희진
- #351 
  - https://github.com/dsrvlabs/crescent-project/issues/47
  - https://github.com/dsrvlabs/assetmantle-project/issues/21
  - https://github.com/dsrvlabs/provenance-project/issues/55
- #360 
  - #361 
- #362 
- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/28

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz-proto/issues/14
   - https://github.com/dsrvlabs/vatz-proto/pull/16

**Reply by xellos00** [reply]
- VATZ bot requires new role to merge or push. 
   - if there's no other way, I will make decision for the rest of process.

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/cosmos-project/issues/122
- https://github.com/dsrvlabs/stargaze-project/issues/26
- https://github.com/dsrvlabs/injective-project/issues/47
- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/27

**Comment by rootwarp** [comment]
# 준교

- https://github.com/dsrvlabs/vatz/issues/322
  - https://github.com/dsrvlabs/vatz/pull/355
- https://github.com/dsrvlabs/vatz/issues/350

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/353
  - https://github.com/dsrvlabs/vatz/pull/364
- https://github.com/dsrvlabs/vatz/issues/351
  - https://github.com/dsrvlabs/agoric-project/issues/145

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/sei-project/issues/48

**Comment by qkrybin** [comment]
## 영빈

- #356 
- #365 
- #325
